src/test/kotlin/com/cjbooms/fabrikt/generators/MicronautControllerGeneratorTest.kt

```diff
@@ -34,6 +34,8 @@ class MicronautControllerGeneratorTest {
         "singleAllOf",
         "pathLevelParameters",
         "parameterNameClash",
+        "responsesSchema",
```

Comment:
**suggestion:** Personally, I think `responseSchema` sounds better. Same for `requestSchema`

Reply:
I'm with you. But "responsesSchema" was already there so I wanted to adapt to the existing style rather than "enforce" my own preference.

Reply:
I understand. Thank you for the explanation